### PR TITLE
simplify issue templates

### PR DESCRIPTION
**Pull request by multun**
These templates have a number of issues:
- the templates assume the only developers working on the project work at SNCF Réseau
- they are deeply unusual. I regularly interact with other bug trackers, and have not seen anything like it, big tech included
- their complexity can scare away contributors
- they don't reflect how SNCF devs actually work. I'm not aware of anyone actually using these templates as intended
- things work just fine, and people only use the description and acceptance criteria part of the templates

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/question.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 name: Question
-about: Ask a question about how the software works, or how to use it
+about: Ask for help with anything related to OSRD
```
Comment (anisometropie) [inline .github/ISSUE_TEMPLATE/question.md]: Maybe keep some of the specifics while opening the possibilities : 

“Ask us for help with how the software works, how to use it or anything else related to OSRD”

File: .github/ISSUE_TEMPLATE/user_story.md [hunk 2]
```diff
@@ -1,60 +1,20 @@
 ---
 name: User story
-title: My user story
-about: An user story is a feature from the point of view of the end user
+about: An user story is an end-user feature
```
Comment (Khoyo) [inline .github/ISSUE_TEMPLATE/user_story.md]: A user story

**Review by anisometropie — DISMISSED** [review]

**Review by eckter — DISMISSED** [review]

**Review by Khoyo — DISMISSED** [review]
LGTM

File: .github/ISSUE_TEMPLATE/user_story.md [hunk 3]
```diff
@@ -1,60 +1,20 @@
...
-
 ## Description
 
-<!-- Please give a brief description of the User Story. What must the user be able to do ? -->
-
-<!-- Is there an existing mockup or wireframe for this US ? If so, please link it here. -->
+**Who is this feature for?**
 
-## Goal
+**What should the user be able to achieve?**
```
Comment (axrolld) [inline .github/ISSUE_TEMPLATE/user_story.md:11]: I would add this paragraph, which would be the equivalent of Goal :
`**Why is this feature necessary ?**`
If we can't answer that question that means we might be working on something pointless, so to me it is an interesting question when creating a new US.

**Review by axrolld — COMMENTED** [review]
Great ! It was about time to clean those !

Don't be surprised if a new template pops up in the coming days, we agreed on creating a sort of backlog of small improvements by creating issues with a specific tag, as we do with the `kind: bug` tag. This is just a proposition so far though.

**Review by El-Youssef — DISMISSED** [review]

**Review by carolnvg — DISMISSED** [review]

File: .github/ISSUE_TEMPLATE/user_story.md (hunk 3 above)
Comment (Khoyo) [reply .github/ISSUE_TEMPLATE/user_story.md:11]: Maybe merge that with the `**Who is this feature for**` part?

Comment (axrolld) [reply .github/ISSUE_TEMPLATE/user_story.md:11]: Hmm to me the end-user and the necessity of a new feature are two different things

Comment (eckter) [reply .github/ISSUE_TEMPLATE/user_story.md:11]: Not all features have to be strictly necessary

Comment (multun) [reply .github/ISSUE_TEMPLATE/user_story.md:11]: I added `**Why do you think this feature would be valuable?**`

**Review by axrolld — APPROVED** [review]

**Review by eckter — APPROVED** [review]

**Pull request merged**